app/models/loan.rb

```diff
@@ -149,64 +137,50 @@ def signing_date_long
     I18n.l(self.signing_date, format: :long, default: '')
   end
 
-  # def short_description
-  #   self.translation('ShortDescription')
-  # end
-  # def description
-  #   self.translation('Description')
-  # end
-
   def coop_media(limit=100, images_only=false)
```

Comment:
I was thinking we might want to replace these with keyword args, but not a big deal for now

Reply:
Yeah, I might do that anyway when I make my revision pass on this PR, I had left them alone at first because I was trying to get the minimal set of changes to get stuff working again, figuring we could worry about refactors later if we wanted them.